docs/api/en/renderers/WebGLRenderer.html

```diff
@@ -249,6 +249,10 @@ <h3>[property:Integer shadowMap.type]</h3>
 		<p>Defines shadow map type (unfiltered, percentage close filtering, percentage close filtering with bilinear filtering in shader)</p>
 		<p>Options are THREE.BasicShadowMap, THREE.PCFShadowMap (default), THREE.PCFSoftShadowMap and THREE.VSMShadowMap. See [page:Renderer Renderer constants] for details.</p>
 
+		<h3>[property:Number shadowMap.normalOffset]</h3>
+		<p>Defines how much the position used to query shadow maps is offset along the object normal.</p>
+		<p>Increasing this value can be used to reduce shadow acne especially in large scenes. The cost is that shadows may appear distorted.</p>
```

Comment:
Default is 0.

>especially in large scenes

What does "large" mean in this case?

Reply:
"Large" means that there's a large shadow frustum.

Maybe something else could also be written here to explain better why not just use bias instead - bias doesn't help much in case the scene has any geometry that is at a shallow angle compared to geometry, whereas normal offset does.

Reply:
Default is *0*.